### ISSUE TITLE
add section for local SSD

### ISSUE THREAD

**Issue by janetstern** [terraform-google-modules/terraform-docs-samples#449]
### TL;DR

Create a VM with attached local SSD by specifying:
[disk_type](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_instance_template#disk_type) - (Optional) The GCE disk type. Such as "pd-ssd", "local-ssd", "pd-balanced" or "pd-standard".
[disk_size_gb](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_instance_template#disk_size_gb) - (Optional) The size of the image in gigabytes. If not specified, it will inherit the size of its base image. For SCRATCH disks, the size must be exactly 375GB.
[type](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_instance_template#type) - (Optional) The type of GCE disk, can be either "SCRATCH" or "PERSISTENT".

 // Local SSD disk
  scratch_disk {
    interface = "SCSI"
  }

**Comment by msampathkumar** [comment]
@janetstern  Thank you for the request.

We have similar samples for https://github.com/terraform-google-modules/terraform-docs-samples/tree/main/compute disk folder

Can you share some more context to understand what & why?

**Comment by msampathkumar** [comment]
Hi @janetstern this request is addressed at https://github.com/terraform-google-modules/terraform-docs-samples/pull/452

Please re-open if need!
